mslib/msui/topview.py

```diff
@@ -56,6 +56,36 @@
 MULTIPLEFLIGHTPATH = 5
 
 
+class CustomColorDialog(QtWidgets.QColorDialog):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.setOptions(QtWidgets.QColorDialog.DontUseNativeDialog)
+
+        # 22 Distinct colors
```

Comment:
Add a section in 
https://github.com/Open-MSS/MSS/blob/develop/NOTICE about the colors.